aws-redshiftserverless-workgroup/src/main/java/software/amazon/redshiftserverless/workgroup/UpdateHandler.java

```diff
@@ -113,13 +123,24 @@ private ResourceModel getUpdatableResourceModel(ResourceModel desiredModel, Reso
                 .build();
     }
 
-    private GetWorkgroupResponse readWorkgroup(final GetWorkgroupRequest awsRequest,
-                                               final ProxyClient<RedshiftServerlessClient> proxyClient) {
-        GetWorkgroupResponse awsResponse;
-        awsResponse = proxyClient.injectCredentialsAndInvokeV2(awsRequest, proxyClient.client()::getWorkgroup);
+    private boolean isWorkgroupStable(final Object awsRequest,
```

Comment:
same here, let's remove the unused param awsRequest?

Reply:
isWorkgroupStable check is performed after multiple steps in update handler i.e during update tags, update workgroup instance so it makes more sense to use object class here